Add documentation for using Google Analytics with Chrome Web Store (#6799)
GoogleChrome/developer.chrome.com: site/en/docs/webstore/google-analytics/index.md

---
layout: "layouts/doc-post.njk"
title: "Using your Google Analytics account with the Chrome Web Store"
seoTitle: "Using your Google Analytics account with the Chrome Web Store"
date: 2023-07-11
description: How to use your Google Analytics account with the Chrome Web Store
---

The Chrome Web Store offers integration with Google Analytics. This allows you to see analytics for

> your Chrome Web Store listing as an alternative to the view offered in the Developer Dashboard.

Reviewer: ```suggestion
your Chrome Web Store listing in addition to the view offered in the Developer Dashboard.
```

---
layout: "layouts/doc-post.njk"
title: "Using your Google Analytics account with the Chrome Web Store"
seoTitle: "Using your Google Analytics account with the Chrome Web Store"
date: 2023-07-11
description: How to use your Google Analytics account with the Chrome Web Store
---

The Chrome Web Store offers integration with Google Analytics. This allows you to see analytics for
your Chrome Web Store listing in addition to the view offered in the Developer Dashboard.

If you’re instead looking to track usage of your extension, see
[Using Google Analytics 4][extensions-ga4].

## Opt-in to Google Analytics {: #opt-in }

When viewing your item in the [Developer Dashboard][developer-dashboard], find the Additional
metrics section on the Store listing page. Click "Opt in to Google Analytics".

{% Img src="image/wVNVUJS8Z8O04i1tJKSdsp6nkRQ2/AVqhwuRqmI00g7AVkIDc.png", alt="Opt in UI for Google Analytics in Developer Dashboard", width="800", height="161" %}

Then, head to [https://analytics.google.com/][ga]. You should have access to a new property which
has been named with your extension ID.

## Limits {: #limits }

> Some limits are in place which can not be changed. These include:

Author: Do you mind saying more about this? There aren't any limits that you can change, so I don't think that wording quite works.

> Some limits are in place which can not be changed. These include:

Author: I've tried rewording this, let me know if that looks good :)

- Data retention is set to two months.
- Data de-identification is enabled, which limits access to non aggregated data to prevent tracking
an individual user. For example, data may be withheld if it does not meet system-defined
[thresholds][ga-thresholds].

## Page views {: #page-views }

Each time a user visits your extension listing, you will see a page view for the following URL: `/webstore/detail/ext/free/EXTENSION_ID/EXTENSION_NAME`

When a user installs your extension, the web store also generates a page view for a fake
`/track_install` URL.

## Events {: #events }

The Chrome Web Store also sends a number of events to your property:

- [`page_view`][ga-pageview], [`session_start`][ga-sessionstart], [`first_visit`][ga-firstvisit] and

> [`user_engagement`][ga-userengagement]

Author: I'm not aware of anything else.

- `install`: This is a [custom event][ga-custom-events] sent when a user installs your extension, at

> the same time as the `track_install` page view (see [above](#page-views)). It is only sent if the

Author: What would the difference be?

> user actually installs your extension (not if they dismiss the permission dialog for example).

Author: I'll tweak this to make that clearer.

forwarded to Google Analytics. For example, an ad could direct users to the following URL:

[https://chrome.google.com/webstore/detail/action-api-demo/ljjhjaakmncibonnjpaoglbhcjeolhkk?utm_source=ad&utm_medium=cpc&utm_campaign=summer-ad-campaign](https://chrome.google.com/webstore/detail/action-api-demo/ljjhjaakmncibonnjpaoglbhcjeolhkk?utm_source=ad&utm_medium=cpc&utm_campaign=summer-ad-campaign)

You will see the following for the corresponding `page_view` and `install` events:

- Session source: `ad`
- Session medium: `cpc`
- Session campaign: `summer-ad-campaign`

The corresponding first user medium/first user campaign/first user source fields will also be set if
this is the first time the user has visited your extension listing.

{% Aside 'warning' %}
Note: It can take between 24-48 hours for data to be finalized. Before then, you may see events
reported with a blank entry for these fields. This will be corrected as the data is finalized.
{% endAside %}

## Tracking conversions {: #track-conversions }

The install event generated when a user installs your extension can be marked as a conversion event.

> Go to "Admin > Conversions" and choose "New Conversion Event". Enter "install" and click save. The

Reviewer 2: ```suggestion
Go to **Admin**, then **Conversions** and choose **New Conversion Event**. Enter "install" and click **Save**. The
```

Reviewer 2: This is per the corporate style guide and long-standing industry convention.

---
layout: "layouts/doc-post.njk"
title: "Using your Google Analytics account with the Chrome Web Store"
seoTitle: "Using your Google Analytics account with the Chrome Web Store"
date: 2023-07-11
description: The Chrome Web Store offers integration with Google Analytics, which allows you to see analytics for your Chrome Web Store listing in addition to the view offered in the Developer Dashboard.
---

The Chrome Web Store offers integration with Google Analytics. This allows you to see analytics for
your Chrome Web Store listing in addition to the view offered in the Developer Dashboard.

If you’re instead looking to track usage of your extension, see
[Using Google Analytics 4][extensions-ga4].

> 

Reviewer 2: Should this have some background on why developers need to do this?

You can get some information from the [MSA we sent out](https://docs.google.com/document/d/1RHxo1dMGGvFD7x-Ml77cu3CB-LE03EIBCK_SwCA8rU4/edit). Please don't write it like an advertisement for the new shiny (as we did in the MSA). Language like that will just sound weird a year from now.

Author: I don't think so (going to resolve for now), but happy to chat more and update if needed. I specifically didn't include anything like that to avoid making this specific to the current transition. This page hopefully just answers the question "How do I see who's finding my extension" without making any reference to the history.
